tests/noto_fonts_for_android_test.py

```diff
@@ -121,5 +132,25 @@ def test_font_weights():
   assert not errors, ", ".join(errors)
 
 
+def test_font_full_weight_coverage():
+  root = etree.parse(str(_noto_4_android_file()))
+  errors = []
+  for family in root.iter("family"):
+    font_to_xml_weights = collections.defaultdict(set)
+    for font in family.xpath("//font[@path]"): 
+      font_to_xml_weights[_font_path(font)].add(int(font.attrib["weight"]))
```

Comment:
you can stuff the index into the key here so it's a tuple of (path, index)`font_to_xml_weights[(_font_path(font), font.attrib.get("index", -1))].add(int(font.attrib["weight"]))`